### DISCUSSION TITLE
Cockpit license

### DISCUSSION THREAD

**Discussion by kgo4956** [cockpit-project/cockpit#20466] [Q&A] [answered]
Hello,
I have a question about the license LGPL v2.1 used for cockpit and especially for project https://github.com/cockpit-project/starter-kit
To create a new component in cockpit menu, we could clone start-kit and modify it. But if we want to use this component inside a commercial project, are we forced to make this project/component open source or is there another means (use of a shared library)?
Thanks.

**Comment by jelly** [comment, marked as answer]
This also came up in the starter-kit repository recently and might be relevant https://github.com/cockpit-project/starter-kit/issues/857
